src/jit/codegencommon.cpp

```diff
@@ -3891,7 +3891,7 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
 #endif // !_TARGET_ARM_
 
 #if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-        if (regType != TYP_STRUCT)
+        if (!varTypeIsStruct(regType))
```

Comment:
That was already done for the non-unix code when the SIMD changes were made. I have done it again for the code in the unix-related ifdefs.